cmd/artifact/test.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -xe
+
+go build
+./artifact create-task > testenv
+. testenv
+
+artifact upload $TASKID $RUNID public/artifact --input artifact
+artifact download $TASKID $RUNID public/artifact --output artifact-downloaded
+diff artifact artifact-downloaded
```

Comment:
Should we delete this file? I think it isn't used, is it?

Reply:
yep!